tfe/testing.go

```diff
@@ -40,12 +67,76 @@ func testTfeClient(t *testing.T, options testClientOptions) *tfe.Client {
 	return client
 }
 
-// skips a test if the test requires a paid feature, and this flag
-// SKIP_PAID is set.
-func skipIfFreeOnly(t *testing.T) {
-	skip := os.Getenv("SKIP_PAID") == "1"
-	if skip {
-		t.Skip("Skipping test that requires a paid feature. Remove 'SKIP_PAID=1' if you want to run this test")
+func upgradeOrganizationSubscription(t *testing.T, client *tfe.Client, org *tfe.Organization) {
+	if enterpriseEnabled() {
+		t.Skip("Cannot upgrade an organization's subscription when enterprise is enabled. Set ENABLE_TFE=0 to run.")
+	}
+
+	req, err := client.NewRequest("GET", "admin/feature-sets", featureSetListOptions{
+		Q: "Business",
```

Comment:
🤔 what do you think about making this Q param dynamic, by allowing a `tier` param that defaults to `Business`. This way tests written for other feature sets can reuse this function as well.

Reply:
Hmm I'm not sure there'd be a reason to run tests against a Developer (or any lower tier) org instead of a Business org. Upgrading an organization to Business serves as a catch all for any paid feature available on TFC.